src/Pages/Products/Products.css

```diff
@@ -0,0 +1,150 @@
+.main {
+    display: flex;
+    flex-direction: column;
+    
+  }
+  .main-body-sec {
+    overflow-y: auto;
+    display: flex;
+    flex: 1;
+    background-color: var(--white-color);
+  }
+  .filter-sidebar {
+    flex-basis: 20%;
+    overflow-y: auto;
+    box-shadow: 4px -8px 8px var(--shadow-color);
+  }
+  .product-button {
+    justify-content: space-around;
+  }
+  .filter-sidebar-heading {
+    display: flex;
+    align-items: center;
+    font: 1.2rem sans-serif;
+    padding: 0.5rem 1rem;
+    justify-content: space-between;
+  }
+  .filter-price {
+    padding: 2rem 1rem;
+    font-weight: bolder;
+    padding-left: 0;
+  }
+  .price-heading {
+    padding: 1rem 2rem;
+  }
+  .filter-price span {
+    padding: 1rem 2rem;
+    font-weight: bolder;
+    font: 1.2em sans-serif;
+  }
+  
+  .range {
+    display: flex;
+    justify-content: space-around;
+    font: 1em sans-serif;
+    color: rgb(92, 90, 90);
+  }
```

Comment:
Try to use global css variables for colors. And also stick to one type of colour convention, so for example only hex codes or onlinesbi rgba

Reply:
thank you for the suggestion!